docs/docs/unit-testing.md

```diff
@@ -132,12 +132,14 @@ const gatsby = jest.requireActual("gatsby")
 module.exports = {
   ...gatsby,
   graphql: jest.fn(),
-  Link: jest.fn().mockImplementation(({ to, ...rest }) =>
-    React.createElement("a", {
-      ...rest,
-      href: to,
-    })
-  ),
+  Link: jest
+    .fn()
+    .mockImplementation(({ activeClassName, activeStyle, to, ...rest }) =>
```

Comment:
Does it make sense to keep this extension? Not all props valid for gatsby-link are valid for a link, so maybe we just grab the `to` prop?

Is there a case where we'd want additional?